{{cookiecutter.project_dirname}}/.env_template

```diff
@@ -1,3 +1,4 @@
+COVERAGE_FILE="._coverage/.coverage"
```

Comment:
A me personalmente non fa impazzire questa riga nel `.env`... in termini pratici, non è propriamente un setting. Non ci sarebbe modo di specificarla altrove?

Reply:
in effetti puó essere trasferita nelle configurazioini di Testing nei settings per renderla piú esplicita

```
 # Coverage
    # https://coverage.readthedocs.io/en/coverage-4.3.3/cmd.html#data-file

    COVERAGE_FILE = os.environ.setdefault("COVERAGE_FILE", "._coverage/.coverage")

```